.travis.yml

```
@@ -15,60 +15,60 @@ jobs:
     # Clang legacy
     - name: Clang-9 Debug
       stage: Legacy
-      env: CXX=clang++-9 BUILD_TYPE=Debug LLVM_VERSION=9
       addons: &clang9
         apt:
           sources:
             - sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-9 main'
               key_url: https://apt.llvm.org/llvm-snapshot.gpg.key
-            - sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main'
-              key_url: https://apt.llvm.org/llvm-snapshot.gpg.key
           packages:
-            - clang-9
             - llvm-9-dev
-            - clang-tidy-10
+            - clang++-9
+            - clang-tidy-9
```

```suggestion
            - clang-tidy-10
```
We should use the same tidy checker for all versions, otherwise, the error/warnings messages get out of sync, e.g., for false positives / false negatives that are fixed in newer versions.